package.json

```diff
@@ -46,6 +46,7 @@
     "p-defer": "^1.0.0",
     "proxy-agent": "^3.1.0",
     "proxy-from-env": "^1.0.0",
+    "rc": "^1.2.8",
```

Comment:
Did we check the license for this dep and its tree? just asking to make sure

Reply:
yes, it is licensed under the two-clause BSD License, MIT License